### PR TITLE
feat: create calling module [AR-1245]

### PR DESCRIPTION
**Pull request by ohassine**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

As we are planning to start implementing  calling, we need to prepare the ground for this feature.

### Solutions

Create new nodule for calling that will hold Avs lib and all calling related stuffs.

Needs releases with:

- [ ] GitHub link to other pull request

### Testing

Manually tested

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: calling/src/androidMain/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.wire.kalium.calling" />
```
Comment (vitorhugods) [inline calling/src/androidMain/AndroidManifest.xml:2]:
```suggestion
<manifest package="com.wire.kalium.calling" /> 

```

File: calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.wire.kalium.calling
+
+actual class Platform actual constructor() {
+    actual val platform: String = "Android ${android.os.Build.VERSION.SDK_INT}"
+}
```
Comment (vitorhugods) [inline calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt:5]: Is a `Platform` class really necessary for this module?

**Review by vitorhugods — CHANGES_REQUESTED** [review]

File: calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt (hunk 2 above)
Comment (ohassine) [reply calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt:5]: No, It's automatically created. 
I just left it to keep the module's structure as it is not possible to only push folders

**Review by typfel — APPROVED** [review]

File: calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt (hunk 2 above)
Comment (MohamadJaara) [inline calling/src/androidMain/kotlin/com/wire/kalium/calling/Platform.kt:5]:
```suggestion
}

```

File: calling/src/commonMain/kotlin/com/wire/kalium/calling/Platform.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.wire.kalium.calling
+
+expect class Platform() {
+    val platform: String
+}
```
Comment (MohamadJaara) [inline calling/src/commonMain/kotlin/com/wire/kalium/calling/Platform.kt:5]:
```suggestion
}

```

File: calling/src/jvmMain/kotlin/com/wire/kalium/calling/Platform.kt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package com.wire.kalium.calling
+
+actual class Platform actual constructor() {
+    actual val platform = "JVM"
+}
```
Comment (MohamadJaara) [inline calling/src/jvmMain/kotlin/com/wire/kalium/calling/Platform.kt:5]:
```suggestion
}

```

**Review by MohamadJaara — COMMENTED** [review]
Just EOF newline

**Review by MohamadJaara — APPROVED** [review]
☎️🤙🏼

**Review by vitorhugods — APPROVED** [review]

**Pull request merged**